astrocut/cube_cut.py

```diff
@@ -941,6 +941,9 @@ def cube_cut(
             # Write the TPF
             tpf_object.writeto(target_pixel_file, overwrite=True, checksum=True)
 
+        if fits_options.get("use_fsspec"):
+            cube._file._file.fs.clear_instance_cache()
```

Comment:
It's a little unfortunate that this involves nested "private" access, but if there's no easy public way to retain public access to that object, then I'm all for it.  This solves a huge problem and if the access to this changes later on we can deal with it again.